applications/openshift/controller/controller_bind_address/rule.yml

```diff
@@ -0,0 +1,43 @@
+documentation_complete: true
+
+title: 'Ensure Controller bind-address argument is set'
+
+description: |-
+    To ensure the Controller Manager service is bound to secure loopback
+    address, edit the Controller Manager pod specification file
+    <tt>/etc/origin/master/master-config.yaml</tt> on the
+    master node(s) and ensure the correct value for the <tt>bind-address</tt>
+    parameter. For example:
+    <pre>kubernetesMasterConfig:
+      controllerArguments:
+        bind-address:
+          - '192.168.1.101'</pre>
+
+     NOTE: The associated value must be reachable by the rest of the cluster, and by
```

Comment:
@shawndwells will you make this a warning instead of a note?

Reply:
done